### PR TITLE
addon: implement flux helm operator add-on

### LINKED ISSUE #100: Proposal: Flux Helm Operator as an Add-on
**User story:**

As a Firekube user,
I'd like to have Flux Helm Operator as an add-on,
so that I can be able to deploy `HelmRelease` resources OOTB

As a profile developer,
I'd like to see wksctl support `HelmRelease` resources,
so that I can include this kind of resources into my profiles.

/cc @palemtnrider could you ptal for go/no-go

### PR DESCRIPTION
**Pull request by chanwit**
This PR implements an add-on for Flux Helm operator.
The current instructions here: https://github.com/fluxcd/helm-operator-get-started assume that we start Flux in a normal way (without using `wksctl`).
This is a tweaked version of helm operator yaml to work with `wksctl`'s Flux service account.

Fixes #100

### CODE REVIEW

File: addons/flux-helm-op/addon.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "kind": "yaml",
+  "category": "Helm",
+  "name": "Flux Helm Operator",
+  "description": "Operator for deploying Helm Chars via Flux",
```
Comment (jrryjcksn) [inline addons/flux-helm-op/addon.json]: Nit: Chars => Charts

File: addons/flux-helm-op/flux-helm-op.yaml [hunk 2]
```diff
@@ -0,0 +1,425 @@
...
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: tiller
+  namespace: kube-system
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: tiller-cluster-rule
```
Comment (jrryjcksn) [inline addons/flux-helm-op/flux-helm-op.yaml]: Was this intended to be `tiller-cluster-role`?

File: addons/flux-helm-op/addon.json (hunk 1 above)
Comment (chanwit) [reply addons/flux-helm-op/addon.json]: fixed

**Comment by chanwit** [comment]
thank you @jrryjcksn for reviewing this!

**Pull request merged**